guides/gatherers.md

```diff
@@ -20,13 +20,47 @@ Here's a collection of build-in gatherers, with information about how to use the
 
 | name                             | implementation                                                                                                                                                 |
 | -------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`cibadmin`](#cibadmin)          | [trento-project/agent/../gatherers/cibadmin.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/cibadmin.go)    
 | [`corosync.conf`](#corosyncconf) | [trento-project/agent/../gatherers/corosyncconf.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/corosyncconf.go)          |
 | [`corosync-cmapctl`](#corosync-cmapctl) | [trento-project/agent/../gatherers/corosynccmapctl.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/corosynccmapctl.go)          |
 | [`hosts`](#hosts-etchosts)       | [trento-project/agent/../gatherers/hostsfile.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/hostsfile.go)                |
 | [`package_version`](#package_version) | [trento-project/agent/../gatherers/packageversion.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/packageversion.go) |
 | [`sbd_config`](#sbd_config)      | [trento-project/agent/../gatherers/sbd.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/sbd.go)                            |
 | [`systemd`](#systemd)            | [trento-project/agent/../gatherers/systemd.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/systemd.go)                    |
 
+### cibadmin
+
+This gatherer allows accessing Pacemaker's CIB information, the output of the `cibadmin` command more precisely.
+As the `cibadmin` command output is in XML format, the gatherer converts it to map/dictionary type of format, so the fields are available with the normal dot access way.
+Some specific fields, such as `primitive`, `clone`, `master`, etc (find the complete list [here](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/cibadmin.go#L48)) are converted to lists, in order to avoid differences when the field appears one or multiple times. 
```

Comment:
Just a suggestion, but consider using the footnote markdown syntax for this, might look better

Reply:
Not really sure about this...
The footnote goes to the end of the page, and it looks a bit disconnected. If we started using this, I would use it consistenly in other places, not only here
I would go with the simple/straightforward solution.
What do you think?

Reply:
ah... true, I looked at the unrendered markdown and it seemed it was possible to do it per section, but when you render it, it goes indeed to the bottom. Nevermind, if it's not possible then this is good enough

Reply:
Ah, I see you added it. It doesn't look bad, if you like it and want to keep it, maybe add  something like:
```[^1]: Find the complete list of supported cibadmin fields [here](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/cibadmin.go#L48)```

So that it also keeps it's context when read separately.

In any case, just a suggestion :+1:

Reply:
At the end I have reverted the change @rtorrero , I don't really like the big disconnection between the sections